### PR TITLE
[PARIS] Amine

### PR DESCRIPTION
**Pull request by enima2684**

### CODE REVIEW

File: starter-code/index.js [hunk 1]
```diff
@@ -1,11 +1,75 @@
...
+    return this._items;
+  }
+
+  set items(value){
+    this._items = value;
+    this.length = this._items.length;
+  }
+
+  add(item) {
+
```
Comment (ta-web-paris) [inline starter-code/index.js]: you seem to have overcomplicated the add function. 

if you remove this code 
``` 
 if(this.length === 0){
      this.items.push(item);
      this.length++;
      return
    } 
```
all the tests still pass. 

Your code is hard to follow here, a cleaner way to write this function would be 

```
add (items) {
    this.items.push(item);
    this.items = this.items.sort((a, b) => a - b);
    this.length = this.items.length;
}

```

Comment (enima2684) [reply starter-code/index.js]: 👍 Right !
